command/delegate/delegate.go

```diff
@@ -30,12 +30,18 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
-type delegateCommand struct {
-	envfile             string
-	awsPoolfile         string
-	defaultPoolSettings vmpool.DefaultSettings
-	poolManager         *vmpool.Manager
-}
+type (
+	delegateCommand struct {
+		envfile             string
+		awsPoolfile         string
+		defaultPoolSettings vmpool.DefaultSettings
+		poolManager         *vmpool.Manager
+	}
+	setupResponse struct {
```

Comment:
can you put the setupResponse in https://github.com/drone-runners/drone-runner-aws/blob/master/command/delegate/data.go